File: BiWeekly Contest 51/A.cpp
class Solution {
public:
    char get(char c, int len){
        return c + len;
    }

    string replaceDigits(string s) {
        for(int i = 0; i < s.size(); i++){
            if(i & 1){
                s[i] = get(s[i-1], s[i] - '0');
            }
        }
        return s;
    }
};

File: BiWeekly Contest 51/A.py
class Solution:
    def get(self, c, length):
        return chr(ord(c) + length)

    def replaceDigits(self, s: str) -> str:
        nums = list(s)
        for i in range(len(s)):
            if i&1:
                nums[i] = self.get(nums[i-1], int(nums[i]))
        return "".join(nums)